### PR TITLE
operations/munge: add example for when using HTTPS

### PR DESCRIPTION
**Pull request by heitorPB**
Docs for https://github.com/omnivector-solutions/slurm-charms/pull/145

Online at: https://omnivector-solutions.github.io/osd-documentation/testing/operations/munge.html

### CODE REVIEW

File: operations/munge.rst [hunk 1]
```diff
@@ -10,9 +10,18 @@ key.
 
 The etcd server has authentication enabled by default, to protect the Munge
 key. In order to be able to retrieve the Munge key, an etcd account is
-required.  OSD provides Juju action ``etcd-create-munge-account`` on the
+required. OSD provides the Juju action ``etcd-create-munge-account`` on the
 ``slurmctld`` charm to automate the creation of such an account.
 
+.. TODO
```
Comment (dusktreader) [inline operations/munge.rst:16]: I'm not quite sure why this is a TODO...

**Review by dusktreader — CHANGES_REQUESTED** [review]

File: operations/munge.rst (hunk 1 above)
Comment (heitorPB) [reply operations/munge.rst:16]: Implementation is not completed yet in the slurm-charms, so the docs are just a start. There are still some bits to figure out and I wanted to have some notes out of my head to not forget later.

Comment (heitorPB) [reply operations/munge.rst]: @dusktreader I upted the document with the complete text. This should have been marked as `draft` from the beginning, but I forgot to do that. The docs are now ready.

File: operations/munge.rst [hunk 2]
```diff
@@ -133,4 +141,38 @@ To get the Munge key in Python, we suggest using the `etcd3gw
...
+   .. code-block:: python
+
+      client = Etcd3AuthClient(host="10.107.185.126", protocol="https",
+                               username="theusername",
+                               password="aVerySafePassword!")
+
+Configuring HTTPS
+=================
+
+The use HTTPS for the connection, you need to supply the TLS certificates to
```
Comment (julianaklulo) [inline operations/munge.rst:162]:
```suggestion
To use HTTPS for the connection, you need to supply the TLS certificates to
```

**Review by julianaklulo — APPROVED** [review]
Looks good, just found a small typo.

File: operations/munge.rst (hunk 2 above)
Comment (heitorPB) [reply operations/munge.rst:162]: Good catch! Thanks!

**Pull request merged**